envs/example/defaults.yml

```diff
@@ -21,6 +21,7 @@ secrets:
   horizon_secret_key:           asdf
   glance_sync:      ADQ64XUQLUWH75M634RVBLP55RKPGGOWG
   monitor_password: asdf
+  telemetry_secret: asdf
 
```

Comment:
A default value for "mongodb_password" will need to be added in the default.yml too. It is in all.yml only. the deployment should work without the value in all.yml so it is either it is in default.yml or the code can handle missing value.